grafana-dashboards/Kepler-Exporter.json

```diff
@@ -41,7 +42,10 @@
       "type": "row"
     },
     {
-      "datasource": "prometheus",
+      "datasource": {
+        "type": "prometheus",
+        "uid": "P1809F7CD0C75ACF3"
```

Comment:
Since we have been using the "datasource": "prometheus" definition of datasource we should go back to the old one later. Grafana doesn't automatically generalize the dashboard for export. 

For some reason I didn't have much success when using the integrated Share -> "Export for sharing externally" so maybe we should just stick to replace the datasource manualy in the exported dashboard.